jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/server/internal/RFC6455Handshaker.java

```diff
@@ -192,17 +192,17 @@ public boolean upgradeRequest(WebSocketNegotiator negotiator, HttpServletRequest
             extensionStack,
             WebSocketConstants.SPEC_VERSION_STRING);
 
-        // Create the Channel
-        WebSocketChannel channel = newWebSocketChannel(handler, negotiated);
+        // Create the Session
+        WebSocketCoreSession session = newWebSocketCoreSession(handler, negotiated);
```

Comment:
let's call it coreSession as a field.   
There are lots of tests that just call it session...  update those as well if you can.